src/vm/gdbjit.cpp

```diff
@@ -106,13 +107,15 @@ GetTypeInfoFromTypeHandle(TypeHandle typeHandle, NotifyGdb::PTK_TypeInfoMap pTyp
                 else
                 {
                     PTR_BYTE base = 0;
-                    if (!pField->IsRVA())
+                    MethodTable* pMT = pField->GetEnclosingMethodTable();
+                    base = pField->GetBaseInDomainLocalModule(pMT->GetDomainLocalModule(NULL));
```

Comment:
It is strange, but it works (at least with my test samples:))  Probably, you could suggest better way to get base?